ui/public/config.json

```diff
@@ -14,6 +14,8 @@
   "logo": "assets/logo.svg",
   "minilogo": "assets/mini-logo.svg",
   "banner": "assets/banner.svg",
+  "loginPageTitle": "CloudStack",
+  "loginPageFavicon": "assets/logo.svg",
```

Comment:
is it just login page or any page's favicon (when it's first opened)?

Instead of re-declaration - should we just re-inforce the appTitle and logo which are already defined in config.json?

Reply:
@Harshitsoni2000 can you please check the ^^ comment?